app/views/browse/service.html

```diff
@@ -62,50 +62,48 @@
                       <dd>{{service.spec.type}}</dd>
                       <dt>IP:</dt>
                       <dd>{{service.spec.clusterIP}}</dd>
+                      <dt ng-if-start="service.spec.externalName">External Name</dt>
+                      <dd ng-if-end>{{service.spec.externalName}}</dd>
```

Comment:
@jwforres Thanks, updated it to "External Hostname:", image below:

![service-page-fields-external-name](https://cloud.githubusercontent.com/assets/3902875/20812653/5ae66466-b7e0-11e6-8d14-96805ecd9b22.png)